perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -204,54 +204,39 @@ def _get_start_end_times(self, out_json, job_params) -> List[Tuple[int]]:
       KeyError: If RW is not present in any dict in job_params
 
     """
-    # Creating a list of just the 'rw' job parameter. Later, we will
-    # loop through the jobs from the end, therefore we are creating
-    # reversed rw list for easy access
-    rw_rev_list = [job_param[consts.RW] for job_param in reversed(job_params)]
+    # Creating a list of just the 'rw' job parameter.
+    rw_list = [job_param[consts.RW] for job_param in job_params]
 
     global_ramptime_ms = 0
-    global_startdelay_ms = 0
     if consts.GLOBAL_OPTS in out_json:
       if consts.RAMPTIME in out_json[consts.GLOBAL_OPTS]:
         global_ramptime_ms = _convert_value(
             out_json[consts.GLOBAL_OPTS][consts.RAMPTIME],
             consts.TIME_TO_MS_CONVERSION, 's')
-      if consts.STARTDELAY in out_json[consts.GLOBAL_OPTS]:
-        global_startdelay_ms = _convert_value(
-            out_json[consts.GLOBAL_OPTS][consts.STARTDELAY],
-            consts.TIME_TO_MS_CONVERSION, 's')
 
-    next_end_time_ms = 0
-    rev_start_end_times = []
-    # Looping from end since the given time is the final end time
-    for i, job in enumerate(list(reversed(out_json[consts.JOBS]))):
-      rw = rw_rev_list[i]
+    start_end_times = []
+    for i, job in enumerate(list(out_json[consts.JOBS])):
+      rw = rw_list[i]
       job_rw = job[_get_rw(rw)]
       ramptime_ms = 0
-      startdelay_ms = 0
       if consts.JOB_OPTS in job:
```

Comment:
Should we create a method to get RampTime for a particular job?

Reply:
refactored to use _get_global_ramp_time() and _get_job_ramp_time()
verified that unit tests are working fine.